File: 01 arrays/exercise - twoSums.py
# 1. Two Sum
# https://leetcode.com/problems/two-sum/description/

class Solution(object):
  def twoSum(self, nums, target):
    for i in range(len(nums)):
      for j in range(i + 1, len(nums)):
        if (nums[i] + nums[j] == target):
          return [i, j]

    raise Exception("no solution")

File: 06 arrays/exercise - twoSums.cs
// 1. Two Sum
// https://leetcode.com/problems/two-sum/description/

public int[] TwoSums(int[] nums, int target)
{
  for (int i = 0; i < nums.Length; i++)
  {
    for (int j = i + 1; j < nums.Length; j++)
    {
      if (nums[i] + nums[j] == target)
      {
        return new int[] { i, j };
      }
    }
  }
  throw new Exception("No solution");
}